### PR TITLE
Added metadata to examples/plotting/aspect.py

### PR DESCRIPTION
**Pull request by Azaya89**
- [x] issues: fixes #11765 
- [x] tests added / passed

### CODE REVIEW

File: examples/plotting/aspect.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
+''' An example demonstrating how to plot a circle within a rectangle in different aspect scales.
```
Comment (bryevdv) [inline examples/plotting/aspect.py]: Suggest some tweak here, the main point to get across is that circles are always circles -- in pixels -- even if the aspect ratio is not 1-1 

```suggestion
''' This example demonstrates how a circle with a data-space radius appears when plotted
with different aspect scales specified.
```

**Review by bryevdv — CHANGES_REQUESTED** [review]
Thanks @Azaya89 just one suggestion

File: examples/plotting/aspect.py (hunk 1 above)
Comment (Azaya89) [reply examples/plotting/aspect.py]: Done.

File: examples/plotting/aspect.py [hunk 2]
```diff
@@ -1,3 +1,11 @@
+''' This example demonstrates how a circle with a data-space radius appears when plotted with different aspect scales specified.
```
Comment (bryevdv) [inline examples/plotting/aspect.py]: line length (apologies I have not been checking this as consistently as I should) 
```suggestion
''' This example demonstrates how a circle with a data-space radius appears
when plotted with different aspect scales specified.
```

**Review by bryevdv — CHANGES_REQUESTED** [review]
One minor nit then this should be good to merge

File: examples/plotting/aspect.py (hunk 2 above)
Comment (Azaya89) [reply examples/plotting/aspect.py]: Done.

File: examples/plotting/aspect.py [hunk 3]
```diff
@@ -1,3 +1,12 @@
+''' This example demonstrates how a circle with a data-space radius appears
+when plotted with different aspect scales specified.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure, bokeh.plotting.show, bokeh.layouts.layout
+    :refs: :ref:`_ug_basic_layouts`
```
Comment (bryevdv) [inline examples/plotting/aspect.py]: It's confusing unfortunatetely: when *defining* the anchors, Sphinx requires the leading `_`, but when *using* the anchors, it should not be present
```suggestion
    :refs: :ref:`ug_basic_layouts`
```

Comment (Azaya89) [reply examples/plotting/aspect.py]: Thank you for this. I had noticed the difference but I didn't know if it was a reasonable question to ask. I have updated it now.

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @Azaya89 !

**Pull request merged**